### ISSUE TITLE
Received item dont stack

### ISSUE THREAD

**Issue by django1m** [HarithMichael/lj-inventory-withdecay#9]
When some received item it dont stack , it make multiple single item , did you guys have a idea why ?

**Comment by OfficialRDB** [comment]
Got that same thing. Every time I get something of the same item, it puts it in a new slot.

**Comment by gl9ck** [comment]
Yep same problem, even if the Item has the same Quality it dont stacks (when item received)

**Comment by elkie1991gaming** [comment]
same issue

**Comment by django1m** [comment]
any fix ?

**Comment by AscendoTuum86** [comment]
@HarithMichael is this supposed to function to prevent item stacking of different qualities? Unfortunately, it's doing the opposite right now, and items of the same quality separate each and every time you get an item (for activities like mining, recycling, etc...) this is extremely troublesome and leads to people's inventories being full and losing out on items almost immediately after starting.
